pkg/process/checks/enabled_checks_test.go

```diff
@@ -88,6 +89,7 @@ func TestConnectionsCheck(t *testing.T) {
 		cfg, scfg := config.Mock(t), config.MockSystemProbe(t)
 		scfg.SetWithoutSource("network_config.enabled", true)
 		scfg.SetWithoutSource("system_probe_config.enabled", true)
+		flavor.SetFlavor("process_agent")
```

Comment:
same here regarding flavor cleanup.

Reply:
@daniel-taf I don't think you did the cleanup in this one?

Reply:
Nice catch. Fixed